layer/main.py

```diff
@@ -741,11 +743,12 @@ def log(
 
     **Images**
 
-    You can log images to track inputs, outputs, detections, activations and more. We support JPEG and PNG formats.
+    You can log images to track inputs, outputs, detections, activations and more. We support GIF, JPEG, PNG formats.
 
     Accepted Types:
     ``PIL.Image.Image``,
     ``path.Path``
+    ``layer.Image``
```

Comment:
As we are not exposing `layer.Image`'s docs, this new PR's additions will not be very discoverable to the users. Either talk more about it here as part of `layer.log`, or make sure to send a PR for the docs separately in the monorepo.

Reply:
I will create a PR in monorepo